.github/workflows/php-checks.yml

```diff
@@ -6,6 +6,58 @@ on:
   merge_group:
 
 jobs:
+  diff-check:
```

Comment:
did you check if it works as expected?

Reply:
It works!
https://github.com/line/line-bot-sdk-php/actions/runs/10313168121/job/28549497048?pr=604